experiments/templatetags/experiments.py

```diff
@@ -85,26 +111,263 @@ def experiment(parser, token):
     experiment content goes here
     {% endexperiment %}
     
-    If the alternative name is neither 'test' nor 'control' an exception is raised
+    If the alternative name is neither 'test'
+    nor 'control' an exception is raised
     during rendering.
     """
     try:
         token_contents = token.split_contents()
-        experiment_name, alternative, weight, user_variable = _parse_token_contents(token_contents)
+        (
+            experiment_name,
+            alternative,
+            weight,
+            user_variable,
+        ) = _parse_token_contents(token_contents)
 
         node_list = parser.parse(('endexperiment', ))
         parser.delete_first_token()
     except ValueError:
-        raise template.TemplateSyntaxError("Syntax should be like :"
-                "{% experiment experiment_name alternative [weight=val] [user=val] %}")
+        raise template.TemplateSyntaxError(
+            "Syntax should be like: "
+            "{% experiment experiment_name alternative"
+            " [weight=val] [user=val] %}")
 
-    return ExperimentNode(node_list, experiment_name, alternative, weight, user_variable)
+    return ExperimentNode(
+        node_list, experiment_name, alternative, weight, user_variable)
 
 
 @register.assignment_tag(takes_context=True)
 def experiment_enroll(context, experiment_name, *alternatives, **kwargs):
+    return _experiment_enroll(
+        context, experiment_name, *alternatives, **kwargs)
+
+
```

Comment:
same here can we delete this extra space?

Reply:
By PEP8, things at the root of a module should be separated by 2 lines. Linter required it.